README.md

```diff
@@ -58,6 +65,14 @@ To run the tests:
 
     npm test
 
+Note: Brave uses port 8080 to communicate between its client and server sides by default. If you are using port 8080 for something else (e.g. a web proxy) then you can set the environment variable `BRAVE_PORT` to make it use a different one.
+
+e.g.
+`BRAVE_PORT=9001 npm run watch`
+
+`BRAVE_PORT=9001 npm run start`
```

Comment:
Note that this is currently broken though because of an added CSP on the html pages.  Maybe post a new issue for that and reference it from here.